cmd/zed/zed.d/all-syslog.sh

```diff
@@ -6,6 +6,7 @@
 . "${ZED_ZEDLET_DIR}/zed-functions.sh"
 
 zed_log_msg "eid=${ZEVENT_EID}" "class=${ZEVENT_SUBCLASS}" \
-    "${ZEVENT_POOL:+"pool=${ZEVENT_POOL}"}" \
+    "${ZEVENT_POOL_GUID:+"pool_guid=${ZEVENT_POOL_GUID}"}" \
```

Comment:
I had it looking up the pool name in an earlier version of the code, but later decided against it.  The problem was that we would get some events before the pool was fully imported, and so we couldn't resolve the pool name.  The other issue is that each pool name lookup (`zed_guid_to_pool`) does a `zpool get` call, which I thought was a little heavyweight for code that runs on each event.